tests/console/sysstat.pm

```diff
@@ -0,0 +1,80 @@
+# SUSE's openQA tests
+#
+# Copyright 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: test sysstat basic functionalities
+# Maintainer: Sergio Rafael Lemke <slemke@suse.cz>
+
+use base 'consoletest';
+use utils 'zypper_call', 'systemctl';
+use strict;
+use testapi;
+
+sub run {
+    select_console('root-console');
+    zypper_call('in sysstat');
+
+    #systemd service:
+    #first we clear old dated logs:
+    script_run 'rm /var/log/sa/sa*';
+
+    systemctl 'start sysstat.service';
+    systemctl 'stop sysstat.service';
+    systemctl 'restart sysstat.service';
+
+    #compare todays date with the date on the file generated when starting the service/systemd unit:
+    validate_script_output "if [ `date +'%Y%m%d'` -eq `ls -1 /var/log/sa/ | cut -c 3-` ]; then echo date_matched; fi", sub { /date_matched/ };
+
+    #Populate /var/log/sa/`date +'%Y%m%d'`, that data will be used on the next tests
+    script_run '/usr/lib64/sa/sa1 5 5';
```

Comment:
As your test relies on this step, I believe we should at least record if command fails. I would recommend to use assert_script_run here.

Reply:
Thanks for the tips, here is a run with the suggested changes:
http://deathstar.suse.cz/tests/734/

could you have a look, I kept some(I consider relevant) of the comments on some of the tests but if needed they can also be removed.

Reply:
Small thing I've missed initially, other things look fine, so please, squash commits so there are no temp ones, make travis happy and we can merge. Thanks!

Reply:
that commits that you can see here: http://deathstar.suse.cz/tests/737/modules/sysstat/steps/1/src Should I also remove them? thanks!

Reply:
No, it's all fine, I just commented to resolve this discussion, however still, could you please improve PR title and reduce number of commits by e.g. squashing them?